README.md

```diff
@@ -83,6 +91,14 @@ console.log('data', await body.json())
 console.log('trailers', trailers)
 ```
 
+_Note: Once a mixin has been called, the `ReadableStream` the body represents cannot be reused, thus calling additional mixins on `.body`, e.g. `.body.json(); .body.text()` will result in an error `TypeError: unusable` being thrown and returned through the `Promise` rejection._
+
+Should you need to access the `body` in plain-text after using a mixin, the best practice is to use the `.text()` mixin first, and manually parse the text to the desired format.
+
+If the scenario arises where you absolutely must have a copy of the result stream, [`.clone`](https://fetch.spec.whatwg.org/#dom-request-clone) can be used _before_ using a `body` mixin, or [`body.pipeThrough`](https://streams.spec.whatwg.org/#readablestream-pipe-through) with a user-defined stream [§](https://web.dev/fetch-upload-streaming), such as [`TransformStream`]https://nodejs.org/api/stream.html#duplex-and-transform-streams.
```

Comment:
cc @ronag how do you think this can be fixed?

Reply:
What do you mean by fixed?

Reply:
I mean that this refer to `body.pipeThrough()` about the `http.request()` while we do not have a WHATWG stream there.

Reply:
I don't understand what the problem is.

Reply:
Like you suggested above. Mentions to web streams should be removed IMHO.

Reply:
That would really neuter the usefulness of the paragraph. I'd really like to retain something for users to point them in the right direction. Could you suggest an edit here?

Reply:
ReadableStream or anything web stream related is not relevant here. I think maybe you want to put another version of this under fetch?

Reply:
OK, I can remove it. I don't think it belong under `fetch` because users that need to copy the result stream _while using undici_ (and that's the important part here) should have something to reference. I understand that from a maintainer's point of view it doesn't belong, but from a user's point of view, that takes away a resource of understanding.

If we can't canonize it in the docs, would it be acceptable to open a new issue as a "how to,"  paste the contents above in the issue, and then close it? So at the least there'd be something searchable in the repo.

Reply:
>  but from a user's point of view, that takes away a resource of understanding.

I'm sorry bit I'm not sure how it belongs here even from a user's point of view. I find it misleading since there is no web stream in this part of the API.

> If we can't canonize it in the docs, would it be acceptable to open a new issue as a "how to," paste the contents above in the issue, and then close it? So at the least there'd be something searchable in the repo.

Sure...